test/hotspot/jtreg/serviceability/dcmd/vm/EventsTest.java

```diff
@@ -64,9 +68,26 @@ public void run_selected(CommandExecutor executor) {
         output.stdoutShouldNotMatch(buildHeaderPattern("Classes unloaded"));
     }
 
+    public void run_max(CommandExecutor executor) {
+        OutputAnalyzer output = executor.execute("VM.events max=" + MAX);
+        long lines = output.asLines().stream().filter(x -> x.contains("Loading class")).count();
+        Assert.assertTrue(lines == MAX, "There are should be " + MAX + " lines");
+    }
+
+    public void run_max_selected(CommandExecutor executor) {
+        OutputAnalyzer output = executor.execute("VM.events log=load max=" + MAX);
+        long lines = output.asLines().stream().filter(x -> x.contains("Loading class")).count();
+        Assert.assertTrue(lines == MAX, "There are should be " + MAX + " lines");
+        output.stdoutShouldNotMatch(buildHeaderPattern("Events"));
```

Comment:
Could also check the output contains the selected event category like in `run_selected` above.

Reply:
Thanks! done.